app/assets/javascripts/shopify_app/partition_cookies.js

```diff
@@ -0,0 +1,13 @@
+(function() {
+  function setCookieAndRedirect() {
```

Comment:
I don't think this function gets called anymore since it's [in StorageAccessHelper](https://github.com/Shopify/shopify_app/pull/647/files#diff-57a97a7498955129995e2cfe66ab38f5R101) right?